app/api/uploadFile/utils.ts

```diff
@@ -0,0 +1,13 @@
+import { v4 as uuidv4 } from "uuid";
```

Comment:
Use kebab-case for file name.

Reply:
It looks better to put this `./utils/*`.

Reply:
resolved!